doc/profiles.md

```diff
@@ -228,6 +230,17 @@ the correct enviroment variables as documented in [`robotology-superbuild-depend
 ### Check the installation
 Follow the steps in https://github.com/robotology/icub-gazebo#usage and/or https://github.com/robotology/icub-models#use-the-models-with-gazebo to check if the Gazebo-based iCub simulation works fine.
 
+## Ignition
+Support for this dependency is enabled by the `ROBOTOLOGY_USES_IGNITION` CMake option.
+This option is set to `OFF` on all platforms as it is still experimental.
+
+### System Dependencies
+Follow the official instructions to install Ignition on your platform, available at https://ignitionrobotics.org/docs.
+
+Different Ignition distributions can be installed alongside. 
+The projects included in the superbuild might require different distributions. 
+From the superbuild point of view, we currently do not allow enabling projects that only support a specific Ignition distribution, therefore all required distributions have to be found in the system.
```

Comment:
If we support only Conda/Linux for `ROBOTOLOGY_USES_IGNITION`, we should describe here how to install Ignition Gazebo on conda (preferably the version tested in CI), as the official docs of Ignition Gazebo do not cover that the installation in conda.

Reply:
Done in https://github.com/robotology/robotology-superbuild/compare/6a3e8b6b3ff38756781c575274145dc19f5aff8b..b76fbfe3a2dc0529aa5d7e9dc32dcced200f03fa